src/index.js

```diff
@@ -9,60 +11,97 @@ import { persistRaw, persistSanitized, pushChanges } from './history/index.js';
 import sanitize from './sanitizer/index.js';
 import getServiceProviders, { getSanitizers } from './service_providers/index.js';
 import { DOCUMENTS_TYPES } from './documents_types.js';
-
-export async function updateServiceProviderDocument({ serviceProviderId, serviceProviderName, document }) {
-  const { documentType, url, contentSelector, sanitizationPipeline } = document;
-  const logPrefix = `[${serviceProviderName}-${DOCUMENTS_TYPES[documentType].name}]`;
-  console.log(`${logPrefix} Scrape '${url}'.`);
-  const content = await scrape(url);
-
-  const { sha: rawSha, filePath: rawFilePath} = await persistRaw(serviceProviderId, documentType, content);
-  if (rawSha) {
-    console.log(`${logPrefix} Save raw file to '${rawFilePath}'.`);
-    console.log(`${logPrefix} Commit raw file in '${rawSha}'.`);
-  } else {
-    console.log(`${logPrefix} No raw changes, didn't commit.`);
+export default class CGUs extends events.EventEmitter {
+  constructor() {
+    super();
+    this.initialized;
+    this.serviceProvidersManifests;
```

Comment:
I don't understand the effect of these statements.

Reply:
Useless. I converted from simple `let` declarations to a class too quickly